spec/collector_spec.rb

```diff
@@ -123,6 +125,8 @@
         stub_dmi('baseboard-product-name', 'ABC456B+')
         stub_dmi('baseboard-serial-number', '34524623454')
         stub_dmi('chassis-serial-number', '2376482364')
+        stub_dmi('bios-version', '4.23')
+        stub_shellout('ipmitool mc info', fixture('ipmitool_mc_info'))
```

Comment:
just a heads-up that ipmitool can be flakey, might want additional error handling